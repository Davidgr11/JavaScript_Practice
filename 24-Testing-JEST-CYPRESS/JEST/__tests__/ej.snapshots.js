const cliente2 = {
    nombre: 'David',
    saldo: 500,
    tipo: 'Premium'
}

describe('Testing cliente2', () => {
    test('Cliente2 existe', () => {
        expect(cliente2).toMatchSnapshot();
    });
});