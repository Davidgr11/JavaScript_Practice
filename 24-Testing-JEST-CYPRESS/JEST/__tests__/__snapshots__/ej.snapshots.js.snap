// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing cliente2 Cliente2 existe 1`] = `
{
  "nombre": "David",
  "saldo": 500,
  "tipo": "Premium",
}
`;

exports[`Testing cliente2 Cliente2 tiene nombre 1`] = `"David"`;
